### DISCUSSION TITLE
Result of optimize

### DISCUSSION THREAD

**Discussion by lapitskaya38** [kernc/backtesting.py#1059] [Q&A] [answered]
Hi!

I got the optimization results in the file header (better parameters) html:
![image](https://github.com/kernc/backtesting.py/assets/141924361/c0ceb41b-20b9-4807-9ca6-ca43d0545da3)

![image](https://github.com/kernc/backtesting.py/assets/141924361/464c0985-e362-4be7-8736-8aca73712267)

But I don't understand where to take this param from the code after optimization? I want to write them to a file with a variable "stats".

**Comment by eervin123** [comment, marked as answer]
Not sure if you are aware that you can grab the best params from the dictionary. `stats._strategy._params`


Also, if the long filenames are an issue (some systems can't handle them) then you can pass `filename="OptimalStrat.html"` into your plot() method.

@kernc explained that to me a long time ago and it has been very beneficial ever since. 
https://github.com/kernc/backtesting.py/discussions/202#discussioncomment-224183

**Reply by lapitskaya38** [reply]
That's what I need: `stats._strategy._params`

Thanks a lot!
